ibis/client.py

```diff
@@ -221,15 +218,12 @@ def execute(self, expr, params=None, limit='default', async=False,
           Scalar expressions: Python scalar value
         """
         ast = self._build_ast_ensure_limit(expr, limit, params=params)
+        result = self._execute_query(ast, async=async, **kwargs)
```

Comment:
`ast` got confusing with js2py transpiler introduced, we might prefer `query_ast` here.

Reply:
renamed.